### ISSUE TITLE
TV accessory turns into 5 separate accessories on restart of Homebridge

### ISSUE THREAD

**Issue by jyy** [SeydX/homebridge-bravia-tvos#90]
after following the setup, everything seemed to be in working order.  A single TV accessory that allowed me to turn on/off and switch between the 4 HDMI inputs.  however, i restarted Homebridge, and the single TV accessory turned into 5 separate accessories; 1 TV accessory with on/off, no input selection, and non-toggleable accessory for each of the 4 HDMI inputs.
here is my config:
```
{
            "name": "BraviaTVOS",
            "tvs": [
                {
                    "name": "Family Room TV",
                    "ip": "192.168.1.XXX",
                    "mac": "--:--:--:--:--:--",
                    "psk": "fakepsk",
                    "inputs": [
                        "extInput:hdmi"
                    ]
                }
            ],
            "platform": "BraviaOSPlatform"
        }
```

**Comment by jyy** [comment]
here is the debug logging on startup:
```
[11/10/2020, 11:54:35 AM] [BraviaTVOS] Family Room TV: Adding new input source
[11/10/2020, 11:54:35 AM] [BraviaTVOS] Family Room TV: {"name":"HDMI 1","identifier":91,"type":"hdmi1","inputType":3}
[11/10/2020, 11:54:35 AM] [BraviaTVOS] Family Room TV: Adding new input source
[11/10/2020, 11:54:35 AM] [BraviaTVOS] Family Room TV: {"name":"HDMI 2","identifier":92,"type":"hdmi2","inputType":3}
[11/10/2020, 11:54:35 AM] [BraviaTVOS] Family Room TV: Adding new input source
[11/10/2020, 11:54:35 AM] [BraviaTVOS] Family Room TV: {"name":"HDMI 3/ARC","identifier":93,"type":"hdmi3/arc","inputType":3}
[11/10/2020, 11:54:35 AM] [BraviaTVOS] Family Room TV: Adding new input source
[11/10/2020, 11:54:35 AM] [BraviaTVOS] Family Room TV: {"name":"HDMI 4","identifier":94,"type":"hdmi4","inputType":3}
[11/10/2020, 11:54:35 AM] [BraviaTVOS] Family Room TV: Polling tv state
[11/10/2020, 11:54:35 AM] [BraviaTVOS] Family Room TV: {"uri":"extInput:hdmi?port=2","source":"extInput:hdmi","title":"HDMI 2"}
```

**Comment by SeydX** [comment]
@jyy 

If you have previously used another plugin OR the v3 of this plugin with the same TV name, you must remove the TV from HomeKit, rename the TV in the config, restart Homebridge and add the TV again. Afterwards, you need to force close apple home and re open.

If you haven't used any other plugin etc. before or the TV hasn't been in Apple Home yet, then jusr force close apple home and re open

**Comment by jyy** [comment]
today is the first time trying to install this plugin and i have not used any other plugins.  i force closed the home app and that didn't do anything.  i also tried removing the TV, renaming it in config, restart Homebridge, re-add TV.  it looked fine when i re-added but then i tested restarting Homebridge after this and same thing....now it is 5 accessories

**Comment by jyy** [comment]
to get it to show up as a single accessory again i can remove and add the TV without making any Homebridge config changes

**Comment by SeydX** [comment]
@jyy 

It is a external accessory and after each hb restart u need to force class the home app. You dont need to remove it from Apple Home

Just force close the home app and re open should fix the issue after a restart

**Comment by jyy** [comment]
That does not work for me.  The only way to get it back to one accessory is adding again

**Comment by SeydX** [comment]
@jyy 

Which ios version?

**Comment by jyy** [comment]
14.1

**Comment by SeydX** [comment]
@jyy 

This is what i mean

https://imgur.com/gallery/rvo3PbX

**Comment by jyy** [comment]
https://imgur.com/gallery/GBfoo7A

**Comment by SeydX** [comment]
@jyy 

Hmmm.. normally it should „link“ the inputs successfully after force close...

I‘ll check this

**Comment by SeydX** [comment]
@jyy 

Is it possible to update to iOS 14.2?

**Comment by jyy** [comment]
yes, i will try that now.

**Comment by SeydX** [comment]
thx

**Comment by jyy** [comment]
Ah that fixed it!  Thanks!  Sounds like Home app in 14.1 isn’t able to regroup.
